daml-script/test/upgrades/coin-v2/Coin.daml

```diff
@@ -7,13 +7,21 @@ template Coin
   with
     issuer : Party
     owner : Party
+    obs : [Party]
   where
     signatory issuer, owner
+    observer obs
 
     nonconsuming choice V2Choice : ()
       controller owner
       do pure ()
 
+
+    choice V2Steal : ContractId Coin
```

Comment:
we should also add this choice in `daml-script/test/upgrades/coin-v3/Coin.daml` since (once we have it) the upgrades check will require that all the choices in V2 are present in V3